pipeline/custom_jenkins_jobs/Jenkinsfile-ceph-deployment.groovy

```diff
@@ -0,0 +1,169 @@
+suite_defs_pacific = [
```

Comment:
@manasagowri @psathyan 
Before moving entire groovy files her, I think we can have a discussion on few things,

- About custom should take test-suite, configuration and every possible thing from user and just create platform to execute on jenkins server.
   **Reason:** 
   - Since most of the test suites are in automated pipeline, I'm assuming we will not miss out any test suites. so user can run his suite by providing all details.
   - This reduces maintenance of updating groovy files all the time.
- [Not a high priority one] And few of the test suites are not been used in the automated pipeline, we need to have plan to move it to some other place.

Reply:
Done.